CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 - Remove slippage from MangroveOrder - resting order now posted at same price
 - Introduce **reserve** hook on MangroveOffer which replaces other reserve logic
 - Add collectByImpersonation to MgvCleaner
+- `ILiquidityProvider` provide simple `updateOffer` and `newOffer` public functions, using default values for `gasprice` and `gasreq`. This unifies interface between Forwarder and Direct strats. Strat builders can still implement a public offer management that lets offer owners set gasreq and gasprice
```

Comment:
I just release a new package... so I guess this will be part of the next package. Do you want it earlier? Then I can release another package. This doesn't change deployed contracts, right?